src/averageAges.js

```diff
@@ -15,6 +15,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const menArrNoCentury = people
+    .filter(person => person.sex === 'm')
+    .map(man => man.died - man.born);
+
+  const menArrWithCentury = people
+    .filter(person => person.sex === 'm'
+    && Math.ceil(person.died / 100) === century)
+    .map(man => man.died - man.born);
```

Comment:
you basically do the same thing in this two variables. i'd suggest that you first filter out all man, and then, if needed, filter out a needed century

Reply:
resolve the first 2 tasks in new way